unit-tests/src/test/scala/java/lang/ProcessUtils.scala

```diff
@@ -9,4 +9,6 @@ object ProcessUtils {
 
   val resourceDir =
     s"${System.getProperty("user.dir")}/unit-tests/src/test/resources/process"
+
+  val bashExtraPaths = List("/bin", "/usr/bin", "/usr/local/bin")
```

Comment:
Given that we are well off into the land of multiple distributions, it
iight be nice to have some comment/breadcrumg for how this apparently mongrel lot was painstakingly derived. 

Not that I am advocating it/them but I do not see a `sbin`. I also do not
see any `snap`.   

On Ubuntu (20.10) before I muck with the PATH for my process, I see
something like `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin`.  Other distributions may have something else.

Thank you for considering my concern and feel free to let me know
that I am off base and lack comprehension.

Reply:
As said above, this is just for bash at the moment. AFAIK Linux distributions have it in `/bin` and BSD variants in `/usr/local/bin`, I don't think shells are usually installed in `/sbin` or other paths.